03/app.js

```diff
@@ -1,14 +1,29 @@
-console.log('DOM');
-
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+	attr: {
+		className: 'btn',
+		title: 'super button',
+	},
+	css: {
+		border: '1px solid #336699',
+		padding: '5px 20px',
+		color: '#444',
+	},
+	text: 'Click me!',
+};
+
+const sectionParent = document.querySelector('.parent-for-button');
+
+const btn = document.createElement('button');
+
+btn.setAttribute('class', buttonSettings.attr.className);
+btn.setAttribute('title', buttonSettings.attr.title);
+
+for (const item in buttonSettings) {
```

Comment:
Tak skonstruowana pętla nie wiele nam daje, lepiej było ją zrobić dla np. `for (const item in buttonSettings.css) {`